dbt_semantic_interfaces/type_enums/node_type.py

```diff
@@ -0,0 +1,9 @@
+from dbt_semantic_interfaces.enum_extension import ExtendedEnum
+
+
+class NodeType(ExtendedEnum):
```

Comment:
Nit: There are a few different nodes in DSI / MF - can you name this more specific?

Reply:
+1. Maybe call this SemanticManifestNodeType or something?

Reply:
Done! 🙂